components/brave_wallet/browser/pref_names.h

```diff
@@ -105,6 +105,9 @@ inline constexpr char kBraveWalletEip1559ForCustomNetworksMigrated[] =
 // 06/2024 to mark existing compressed Solana NFTs as compressed.
 inline constexpr char kBraveWalletIsCompressedNftMigrated[] =
     "brave.wallet.is_compressed_nft_migrated";
+// Added 11/2024 to mark Aurora mainnet as migrated.
+inline constexpr char kBraveWalletCustomNetworksAuroraMainnetMigrated[] =
+    "brave.wallet.custom_networks.aurora_mainnet_migrated";
 
```

Comment:
Let's avoid being it a netsted pref of `brave.wallet.custom_networks` since it's already a separate pref
Just `brave.wallet.aurora_mainnet_migrated` would be enough

Yes, there is `brave.wallet.custom_networks.fantom_mainnet_migrated` above but such approach causes problems when fromat of parent pref gets changed.